public/robots.txt

```diff
@@ -1,3 +1,3 @@
 # http://www.robotstxt.org
 User-agent: *
-Disallow:
+Disallow: /
```

Comment:
I'm worried we'll forget to remove this when we merge to master. Is there an alternative to robots.txt to block this domain (like an HTTP header we can configure for the Netlify app?), or should we do something to make really sure we remember?

Reply:
I think this is best for now and don't worry I won't miss this when it comes to merging 😉

Reply:
I am tracking our crimes in the main Octane PR so we don’t forget to undo them